gradle.properties

```diff
@@ -24,3 +24,6 @@ systemProp.org.gradle.warning.mode=fail
 # forcing to use TLS1.2 to avoid failure in vault
 # see https://github.com/hashicorp/vault/issues/8750#issuecomment-631236121
 systemProp.jdk.tls.client.protocols=TLSv1.2
+
+# jvm args for faster test execution by default
+systemProp.tests.jvm.argline=-XX:TieredStopAtLevel=1
```

Comment:
does this have any security implications? I know it's just on the tests and not applied in production; just wondering if there are any latent security side effects?

Reply:
no. java uses tiered compilation by default: starting at interpreter, then when code gets hot, it transitions back and forth through tiers. your code without this parameter would transition from level 1 to higher levels (e.g. recompiling with profiling hooks, recompiling with c2, etc). This just tells it to stop at level 1. We do the same thing with lucene builds.

Reply:
awesome! Thank you for the explanation!